### ISSUE TITLE
A few suggestions

### ISSUE THREAD

**Issue by Chocolateiro** [kanryu/quickviewer#37]
Here are a few features I wish I had in QuickViewer:
1. Be able to hide scroll bars when zooming in full screen. Besides the error mentioned [here](https://github.com/kanryu/quickviewer/issues/8#issuecomment-313066485) persists. (The horizontal bar is displayed even if it is not necessary)
2. Show relative file path in the window title
3. Be able to move the enlarged image up/down with the respective up/down keys
4. Do not save history
5. Be able to open compressed files with password

**Comment by kanryu** [comment]
1. This is the behavior of the Qt standard. There is a possibility to be able to change.
1. The application may be opening files on another drive or PC. In that case it will be useless. We can display only the folder name instead.
1. We will need to change the behavior of not only up and down, but also the left and right keys. Currently the left and right keys are switched to the next or previous image irrespective of whether they are enlarged or not, but it will be necessary to set to the operation to change the display position in the image by changing it.
1. possible
1. It is possible, but it will take some time as it will have to be tested in all of the several archive formats.

**Comment by Chocolateiro** [comment]
I would be happy with any of these implementations.

**Comment by kanryu** [comment]
1. completed
1. completed (add mode to show volume name only on window title)
1. completed (can scroll image with cursor keys when images bigger than the window)
1. completed
1. no work (Since it has low priority as a function to be implemented by the image viewer, it is not implemented this time.)

**Comment by kanryu** [comment]
@Chocolateiro I made it :)

**Comment by Chocolateiro** [comment]
Wonderful. I appreciate your work.

I've tried the new version, but I have these drawbacks.
The problem with unnecessary scroll bars has been solved, but the bars are still displayed in full screen mode.
Why not display the full file name in addition to the location?
Is it possible to change the previous/next image with the same scroll keys at the end of the image?
Is it possible to hide the page bar permanently?
